mqt/qcec/symbolic.py

```diff
@@ -0,0 +1,147 @@
+"""Functionality for checking equivalence of parameterized quantum circuits."""
```

Comment:
As discussed in person, since parameters don't mean quite the same for quiskit and mqt. I'm going to at least streamline the parts that are exposed to the user (like `n_parameterized_checks` vs `n_symbolic_checks`)